README.md

```diff
@@ -40,6 +40,12 @@ firewall rules may not be able to contact the Charm Store. In these network
 restricted  environments the resource can be uploaded to the model by the Juju
 operator.
 
+## Deploying on LXD
+
+When deploying on LXD juju will deploy the lxd profile found inside of this charm, renaming it in the process this profile can be inspected by running `lxd profile list` and choosing the profile matching your juju model name, once you have this you can run `lxc profile show ${PROFILE_NAME}`.
```

Comment:
@VariableDeclared, there's no prize for longest sentence ;)

That aside, I'm not sure charm operators need to know this level of detail about the embedded profile. @evilnick, do we have any doc blurbs about cdk on lxd that we could link from here?

Reply:
@kwmonroe No. all the localhost stuff is currently predicated around conjure-up. There won't be any lxd docs until I write them based on however this turns out.

Reply:
I had a suspicion this might be the case. I can cut down the paragraph if required :smile: